### PR TITLE
[A2] Update project description

### PR DESCRIPTION
**Pull request by okg00dbye**
This hopefully gives better clarity on the level of technicality to be expected during the presentation and the focus - which is, to sell the chosen idea as the coolest

### CODE REVIEW

**Comment by chrisgzf** [comment]
@jialin7878 tweaked the way we represent some stuff. do take a look and let me know if there are any issues!

@sooyj need your help to review this - if you are okay with this we can merge it.

**Comment by sooyj** [comment]
Thanks @jialin7878  and @chrisgzf for the added details and updates :-)

File: website/coursework/seminar/index.md [hunk 1]
```diff
@@ -112,19 +119,32 @@ You will also get points if your friends post stuff on your forum posts. The ide
...
+<p>The slides for the presentation should be named <code>{'group-<number>-seminar.{pptx|key}'}</code> and uploaded to Coursemology by <strong>{formatDate(assignments.seminar.slides_submission)}</strong>, because we will be downloading the slides to project on-screen or screen-share during class.</p>
+
+<details>
+  <summary>If you are using Google Slides</summary>
+  <p>
+    Instead of the above instructions, upload a PDF version of your slides <code>{'group-<number>-seminar.pdf'}</code> and upload a text file <code>{'group-<number>-seminar.txt'}</code> containing just the publicly-accessible view-only link to your slides to Coursemology by <strong>{formatDate(assignments.seminar.slides_submission)}</strong>.
+  </p>
+
+  <p>
+    Do not make any changes to your slides after the submission deadline, even if it is just tweaking animations. During the presentation day and right before your team presents, one of your team members should be ready with the revision history and show us that there has been no changes past the deadline. If there are any, we will ask you to rollback the presentation' revision to right before the submission deadline, before your presentation starts.
```
Comment (zhuhanming) [inline website/coursework/seminar/index.md:138]:
```suggestion
    Do not make any changes to your slides after the submission deadline, even if it is just tweaking animations. During the presentation day and right before your team presents, one of your team members should be ready with the revision history and show us that there has been no changes past the deadline. If there are any, we will ask you to rollback the presentation's revision to right before the submission deadline, before your presentation starts.
```

File: website/coursework/seminar/index.md [hunk 2]
```diff
@@ -48,62 +48,69 @@ This assignment is currently being updated. Details are tentative and will be fi
...
-4. What is good, bad and ugly about the application
-5. Suggested improvements
-6. Commercial potential
-7. New app(s)/business models/ideas arising from studying the application
+1. Description of the product
+2. Why we should care about the product you chose
+3. What is good about the product
+4. What are the limitations of the product, and suggestions on how they can be overcome
+5. How the product can be implemented if you had to do it (Describe the program organisation by using high-level views like logic flow, identifying technologies used where possible, etc.)
+6. New app(s)/business models/ideas arising from studying the product
```
Comment (zhuhanming) [inline website/coursework/seminar/index.md:79]:
```suggestion
1. Description of the product
1. Why we should care about the product you chose
1. What is good about the product
1. What are the limitations of the product, and suggestions on how they can be overcome
1. How the product can be implemented if you had to do it (Describe the program organisation by using high-level views like logic flow, identifying technologies used where possible, etc.)
1. New app(s)/business models/ideas arising from studying the product
```

**Review by zhuhanming — COMMENTED** [review]
Nice! I think perhaps can have even greater emphasis on the "convince your classmates that your idea is the coolest" part throughout the write up, especially for the group presentation section.

**Pull request merged**